src/Altinn.Common.PEP/Altinn.Common.PEP/Helpers/DecisionHelper.cs

```diff
@@ -187,12 +199,70 @@ private static List<XacmlJsonAttribute> CreateSubjectAttributes(IEnumerable<Clai
                 {
                     attributes.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.SessionId, claim.Value, DefaultType, claim.Issuer));
                 }
+                else if (IsSystemUserClaim(claim, out SystemUserClaim userClaim))
+                {
+                    systemUserAttribute = CreateXacmlJsonAttribute(AltinnXacmlUrns.SystemUserUuid, userClaim.Systemuser_id[0], DefaultType, claim.Issuer);
+                }
+                else if (IsUserIdClaim(claim.Type))
+                {
+                    userIdAttribute = CreateXacmlJsonAttribute(AltinnXacmlUrns.UserAttribute, claim.Value, DefaultType, claim.Issuer);
+                }
+                else if (IsPersonUuidClaim(claim.Type))
+                {
+                    personUuidAttribute = CreateXacmlJsonAttribute(AltinnXacmlUrns.PersonUuidAttribute, claim.Value, DefaultType, claim.Issuer);
+                }
+                else if (IsPartyIdClaim(claim.Type))
+                {
+                    partyIdAttribute = CreateXacmlJsonAttribute(AltinnXacmlUrns.PartyAttribute, claim.Value, DefaultType, claim.Issuer);
+                }
+                else if (IsResourceClaim(claim.Type))
+                {
+                    partyIdAttribute = CreateXacmlJsonAttribute(AltinnXacmlUrns.ResourceId, claim.Value, DefaultType, claim.Issuer);
+                }
+                else if (IsOrgAttributeClaim(claim.Type))
```

Comment:
OrgAttribute betyr for meg: `urn:altinn:org`
så ungå helst org forkortelsen når det menes organizationnumber

Reply:
Agree. Fixed